.ci/checkDeletedTxt.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. "$SCRIPT_DIR/helper.sh"
+cd "$SCRIPT_DIR"
+
+MASTER="master"
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "$BRANCH" = "$MASTER" ]];then
+    echo "ok - Can only check deleted.txt when not on '$MASTER' branch"
+    exit 0
+fi
+
+main() {
+    local IFS=$'\n'
+    declare -a deletions=()
+    local failed=0
+    # Find all deleted js files
+    deletions+=($(git diff-index --diff-filter=D --name-only $MASTER))
+    if (( ${#deletions[@]} > 0 ));then
+        for f in "${deletions[@]}";do
+            local id=$(git show $MASTER:"$f"|grepTranslatorId)
+            if ! grep -F "$id" '../deleted.txt';then
```

Comment:
We only should look here at delete translators and ignore other files which might get deleted. Thus, I suggest to add here a test
```bash
if [[ -n "$id" ]];then
```